web/src/components/core/Sidebar.jsx

```diff
@@ -129,21 +130,30 @@ export default function Sidebar ({ children }) {
   return (
     <>
       <AppActions>
+        <If
+          condition={notification.issues}
+          then={
+            <Button
+              aria-label={_("Show issues popup")}
+              variant="warning"
+              icon={<Icon name="warning" />}
+              onClick={() => setShowWarningPopup(prev => !prev)}
```

Comment:
`onClick={() => setShowWarningPopup(true)}` is probably better, since this is always used to open the popup and can't be used to close it.